data/hai/hai missions.txt

```diff
@@ -81,6 +81,24 @@ mission "First Contact: Unfettered"
 		attributes "unfettered"
 	on offer
 		conversation
+			branch hai
+				has "First Contact: Hai: offered"
+				
+			`This planet is populated by an alien species that resembles giant, intelligent squirrels. Do you want to approach one of them?`
+			choice
+				`	(Sure.)`
+				`	(Not now, it's too risky.)`
+					defer
```

Comment:
This "too risky" choice doesn't really make sense in this context. The player knows nothing of the Hai at this point. Why would it be risky to talk to these Hai and not the other Hai (as the choice for the Hai first contact just has a "not now" option).

Since the player at least knows that the Unfettered are hostile, perhaps mention that in the introduction.
```suggestion
			`This planet is populated by a hostile alien species that resembles giant, intelligent squirrels. Do you want to approach one of them?`
			choice
				`	(Sure.)`
				`	(Not now, it's too risky.)`
					defer
```

Reply:
I like that change, lets go with that.